documentation/book/ref-metrics-alertmanager-examples.adoc

```diff
@@ -0,0 +1,48 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-metrics-prometheus-alertmanager.adoc
+[id='ref-metrics-alertmanager-examples-{context}']
+
+= Alerting rule examples
+
+Example alerting rules for Kafka and Zookeeper metrics are provided with {ProductName} for use in a xref:proc-metrics-deploying-prometheus-{context}[Prometheus deployment].
+
+The rules are defined in a `PrometheusRule` resource in the link:https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/install/prometheus-rules.yaml[`prometheus-rules.yaml`^] file
+
+.Kafka alerting rules
+
+* `UnderReplicatedPartitions`: gives the number of partitions for which the current broker is the lead replica but the follower replicas are behind.
+This metric provides insights about offline brokers that host the follower replicas.
```

Comment:
They're not necessarily offline. They may have been off line and are still catching up. The interbroker replication may be throttled, etc.
```suggestion
This metric provides insights about brokers that host the follower replicas. 
Those followers are not keeping up with the leader.
Reasons for this could include being (or having been) offline, and over-throttled interbroker replication.
```